client/a8c-for-agencies/sections/purchases/licenses/license-preview/use-license-actions.ts

```diff
@@ -82,6 +83,13 @@ export default function useLicenseActions(
 				isExternalLink: true,
 				isEnabled: licenseState === LicenseState.Attached,
 			},
+			{
+				name: translate( 'Upgrade' ),
+				href: `/marketplace/hosting/wpcom`,
```

Comment:
We can replace this with `A4A_MARKETPLACE_HOSTING_WPCOM_LINK` constant

Reply:
Thanks, I updated it!